### PR TITLE
WIP Update

### PR DESCRIPTION
**Pull request by acidmanifesto**

### CODE REVIEW

File: src/AoeLoot.cpp [hunk 1]
```diff
@@ -15,6 +15,155 @@
...
+        // Announce Module
+        if (AOELootEnable)
+        {
+            if (AOELootAnnounceModule)
+            {
+                ChatHandler(Player->GetSession()).SendSysMessage("This server is running the |cff4CFF00AOE Loot Azerothcore Custom |rmodule.");
+            }
+        }
+    }
+};
```
Comment (Winfidonarleyan) [inline src/AoeLoot.cpp:78]: why need this?

File: src/AoeLoot.cpp [hunk 2]
```diff
@@ -15,6 +15,155 @@
...
+            // Load Configuration Settings
+            SetInitialWorldSettings();
+        }
+    }
+
+    // Load Configuration Settings
+    void SetInitialWorldSettings()
+    {
+        AOELootEnable = sConfigMgr->GetBoolDefault("AOELoot.Enable", 1);
+        AOELootAnnounceModule = sConfigMgr->GetBoolDefault("AOELoot.Announce.Enable", 1);
```
Comment (Winfidonarleyan) [inline src/AoeLoot.cpp:59]:
```suggestion
        AOELootEnable = sConfigMgr->GetOption<bool>("AOELoot.Enable", 1);
        AOELootAnnounceModule = sConfigMgr->GetOption<bool>("AOELoot.Announce.Enable", 1);

```

File: src/AoeLoot.cpp (hunk 1 above)
Comment (acidmanifesto) [reply src/AoeLoot.cpp:78]: It was to test that i understood using the hooks right really. It did work and can be removed if you want. Right now i am dealing with a internet outage so my response will be slow.

File: src/AoeLoot.cpp [hunk 3]
```diff
@@ -15,6 +15,156 @@
...
+        }
+    }
+};
+// Group all custom scripts
+void AddSC_AoeLoot()
+{
+    new AoeLoot_World();
+    new AoeLootAnnounce();
+    new AoeLoot_Player();
+}
```
Comment (pangolp) [inline src/AoeLoot.cpp:170]: Here a line break is missing at the end of the file.

**Comment by pangolp** [comment]
> Be advise we are still working modulating this feature. It currently is not working until we get the hooks implemented.

I was only trying to help, I think the idea is very good.

**Comment by acidmanifesto** [comment]
> > Be advise we are still working modulating this feature. It currently is not working until we get the hooks implemented.
> 
> I was only trying to help, I think the idea is very good.

Oh i agree and will drop it in when im free. Been having people message me that its not working not realizing its wip for az

**Review by pangolp — APPROVED** [review]

**Pull request closed without merging**